lib/paypal_express/api.rb

```diff
@@ -24,13 +24,13 @@ def gateway
 
       def verify_credentials
         unless paypal_login.present? && paypal_password.present? && paypal_signature.present? then
-          raise "Please ensure all Paypal Credentails are set in your env file." 
+          raise "Please ensure all Paypal Credentails are set in your env file."
         end
       end
 
       # DEFAULT value for test mode is true.
       def test_mode
-        FlexCommerceApi.config.order_test_mode || true
```

Comment:
Previously this line would always return true as in production `FlexCommerceApi.config.order_test_mode` is set to false which then defaults to use `true`.